components/StyledComponents.js

```diff
@@ -10,4 +10,9 @@ export const FlexContainer = styled.div`
 export const FlexChild = styled.div`
   flex: 1;
   margin: 10px;
-`;
\ No newline at end of file
+`;
+
+function TodoList() {
+}
+
+export default TodoList;
```

Comment:
It's better not to leave empty components in the code. Remove them until you're ready to implement them.